packages/cuba-react-ui/src/ui/paging/Paging.tsx

```diff
@@ -0,0 +1,123 @@
+import {PaginationConfig} from "antd/lib/table";
+import {DataCollectionStore} from "@cuba-platform/react-core";
+import * as QueryString from "querystring";
+import React from "react";
+import {Pagination} from "antd";
+import {toJS} from "mobx";
+
+type Props = {
+  paginationConfig: PaginationConfig;
+  total?: number,
+  onPagingChange: (current: number, pageSize?: number) => void;
+}
+
+/**
+ * Paging defaults
+ */
+export const defaultPagingConfig: PaginationConfig = {
+  current: 1,
+  pageSize: 10,
+  defaultPageSize: 10,
+  showSizeChanger: true,
+  pageSizeOptions: ['10', '20', '30']
+};
+
+/**
+ * Ant Design Pagination component adopted to our List and DataCollectionStore
+ */
+export class Paging extends React.Component<Props> {
+
+  render() {
+    const {paginationConfig, total} = this.props;
+
+    return (
+      <Pagination
+        {...paginationConfig}
+        total={total}
+        pageSizeOptions={toJS(paginationConfig.pageSizeOptions)}
+        onChange={this.props.onPagingChange}
+        onShowSizeChange={this.props.onPagingChange}
+      />
+    );
+  }
+}
+
+/**
+ * Returns new PaginationConfig object with 'current' and 'pageSize' params, parsed from location.search.
+ * If param not found in search url, previous value (currentPrev, pageSizePrev) will be returned.
+ *
+ * @param locationSearch trying to find paging params here
+ * @param currentPrev - previous value, will be returned in result if new value not found in locationSeach
+ * @param pageSizePrev - previous value, will be returned in result if new value not found in locationSeach
+ */
+export function parsePagingParams(locationSearch: string, currentPrev: number | undefined, pageSizePrev: number | undefined): PaginationConfig {
+  // remove '?' from start of location.search
+  const locationSearchParams = locationSearch && locationSearch.length > 0 ? locationSearch.substr(1) : '';
```

Comment:
do we need to remove it manually?

from query string docs:

```
console.log(location.search);
//=> '?foo=bar'
 
const parsed = queryString.parse(location.search);
console.log(parsed);
//=> {foo: 'bar'}
```

Reply:
Can't find anything about question mark
https://nodejs.org/api/querystring.html